tests/test_l2_basic_dd.py

```diff
@@ -0,0 +1,71 @@
+import pytest
+from ptf.testutils import simple_tcp_packet, send_packet, verify_packets
+
+@pytest.fixture(scope="module", autouse=True)
+def discovery(npu):
+     npu.objects_discovery()
+
+def test_l2_trunk_to_trunk_vlan_dd(npu, dataplane):
+    """
+    Description:
+    Check trunk to trunk VLAN members forwarding
+
+    #1. Create a VLAN 10
+    #2. Add two ports as tagged members to the VLAN
+    #3. Setup static FDB entries for port 1 and port 2
+    #4. Send a simple vlan tag (10) packet on port 1 and verify packet on port 2
+    #5. Clean up configuration
+    """
+    vlan_id = "10"
+    macs = ['00:11:11:11:11:11', '00:22:22:22:22:22']
+
+    cmds = [{
+        "name": "vlan_10",
+        "op": "create",
+        "type": "SAI_OBJECT_TYPE_VLAN",
+        "attributes": [
+            "SAI_VLAN_ATTR_VLAN_ID", vlan_id
+        ]
+    }]
+
+    for idx, mac in enumerate(macs):
+        cmds.append({
+            "name": f"vlan_member_{idx}",
+            "op": "create",
+            "type": "SAI_OBJECT_TYPE_VLAN_MEMBER",
+            "attributes": [
+                "SAI_VLAN_MEMBER_ATTR_VLAN_ID", "$vlan_10",
+                "SAI_VLAN_MEMBER_ATTR_BRIDGE_PORT_ID", f"$BRIDGE_PORT_{idx}",
```

Comment:
Is this an object added to cache during discovery?

Reply:
Thanks @andriy-kokhan I suggest creating a README for tips-and-techniques for things like this, logging, debugging, etc. As questions from users come up, this could grown over time.